appshared/src/main/java/io/reark/rxgithubapp/shared/data/StoreInterface.java

```diff
@@ -0,0 +1,14 @@
+package io.reark.rxgithubapp.shared.data;
+
+import android.net.Uri;
+
+import io.reark.rxgithubapp.shared.pojo.GitHubRepository;
+import io.reark.rxgithubapp.shared.pojo.GitHubRepositorySearch;
+
+/**
+ * Created by ttuo on 26/06/16.
+ */
```

Comment:
You could remove these

https://stackoverflow.com/questions/20109203/android-studio-how-to-remove-update-the-created-by-comment-added-to-all-new-c